### ISSUE TITLE
Documentation request: how to use SerializableReference with SerializableType

### ISSUE THREAD

**Issue by rafaeldolfe** [CareBoo/Serially#78]
In a Unity post you show a screenshot of using the reference with a type.

![image](https://github.com/CareBoo/Serially/assets/42071458/b3158744-7cb0-4946-b7bf-cbfc83381454)

I haven't figured out how to do this in my project. Can you show how this is done in more detail?

**Comment by jasonboukheir** [comment]
Hi, the code for this is found in the documentation:
https://careboo.github.io/Serially/2.2/manual/ShowSerializeReference.html

There is no `SerializableType` in that picture. The `ShowSerializeReference` and `SerializableType` share a similar drawer, which is why they look similar 😉.

**Comment by rafaeldolfe** [comment]
Ah, gotcha.

As a user of this repo, I would personally love for the main parts of the documentation to be available directly in the README. Even better would be to have it showcase the main usages so I don't need to figure out how it can be used. I looked at the docs earlier but thought they might be incomplete not finding anything about the SerializableType interacting with ShowSerializeReference, having thought the Unity forum was hinting at that. 
Even just copy pasting the docs into the README would be nice. Most repos I've used in the past don't use Github pages.

Hope I don't sound ungrateful - it's a really nice package, thanks for making it available open source.
